### PR TITLE
Interpreter shell

### PR DESCRIPTION
**Pull request by jaanonim**

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "typeAcquisition": {
+        "include": [
+            "jest"
```
Comment (n2oneProgrammer) [inline jsconfig.json:4]: Polish?

File: core/run.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Lexer from "./lexer.js";
+
+export default class Run {
```
Comment (n2oneProgrammer) [inline core/run.js]: Name class should be noun. I think the better is Runner

**Review by n2oneProgrammer — CHANGES_REQUESTED** [review]

File: jsconfig.json (hunk 1 above)
Comment (jaanonim) [reply jsconfig.json:4]: no https://jestjs.io/

**Review by n2oneProgrammer — APPROVED** [review]
LGTM

**Review by mwawrzen — APPROVED** [review]
LGTM

**Pull request merged**